Makefile

```diff
@@ -6,3 +6,21 @@ test:
 
 fix:
 	pipenv run pytest -x tests
+
+build:
+	pipenv install -d setuptools wheel
+	pipenv run python3 setup.py sdist bdist_wheel
+
+test_release:
+	pipenv install -d twine
+	pipenv run twine upload --repository testpypi --username __token__ --password ${PYPI_TEST_TOKEN} --non-interactive dist/*
```

Comment:
This is bad because `make` will echo the command back, revealing the token. I would suggest that we use environment variables `TWINE_USERNAME` and `TWINE_PASSWORD` for this task.

Reply:
Ok, I added extra env vars for CircleCI jobs. But I think that currently these variables are masked so token is not revealed. Correct me If I'm wrong please.

Reply:
They are masked in the output on the web interface, but we should not rely on that. I would take things even further and advise you to NEVER put any kind of secret on a command-line because that just builds bad habits.

Reply:
Thanks for clearifying things a little bit more. But can you really always avoid placing secrets on a command-line in CI/CD jobs? I added `TWINE_USERNAME` and `TWINE_PASSWORD` env vars but this seems to have no effect as username is still visible in CircleCI pipline logs.